addons/website_sale_wishlist/static/src/js/website_sale_tour_wishlist.js

```diff
@@ -0,0 +1,118 @@
+odoo.define('website_sale_wishlist.tour', function (require) {
+'use strict';
+
+var tour = require("web_tour.tour");
+var base = require("web_editor.base");
+
+tour.register('shop_wishlist', {
+    test: true,
+    url: '/shop',
+    wait_for: base.ready(),
+},
+    [
+        {
+            content: "click on add to wishlist",
+            trigger: '.o_add_wishlist',
+        },
+        {
+            content: "go to wishlist",
+            extra_trigger: ".label:contains(1)",
+            trigger: 'a[href="/shop/wishlist"]',
+        },
+        {
+            content: "remove Ipad",
+            trigger: 'a.o_wish_rm:first',
+        },
+        {
+            content: "go back to the store",
+            trigger: "a[href='/shop']"
+        },
+        {
+            content: "click on add to wishlist",
+            trigger: '.o_add_wishlist',
+        },
+        {
+            content: "click on login",
+            trigger: 'a[href="/web/login"]',
+        },
+        {
+            content: "login Email",
+            trigger: 'form input[name="login"]',
+            run: "text admin"
+        },
+        {
+            content: "click on login",
+            trigger: "body",
+            run: function (actions){
+                $('.oe_login_form input[name="redirect"]').val("/shop");
+            },
+        },
+        {
+            content: "login password",
+            trigger: 'form input[name="password"]',
+            run: "text admin"
+        },
+        {
+            content: "login admin",
+            trigger: 'form:has(input[name="login"]) .btn-primary',
+        },
+        {
+            content: "go to wishlist",
+            extra_trigger: ".label:contains(1)",
+            trigger: 'a[href="/shop/wishlist"]',
+        },
+        {
+            content: "go back to the store",
+            trigger: "a[href='/shop']"
+        },
+        {
+            content: "search ipad",
+            trigger: 'form input[name="search"]',
+            run: "text ipad retina",
+        },
+        {
+            content: "search ipad retina",
+            trigger: 'form:has(input[name="search"]) .oe_search_button',
+        },
+        {
+            content: "select ipad",
+            trigger: '.oe_product_cart a:contains("iPad Retina Display")',
+        },
+        {
+            content: "select ipad 32GB",
+            extra_trigger: '#product_detail',
+            trigger: 'label:contains(32 GB) input',
+        },
+        {
+            content: "click on add to wishlist",
+            trigger: '#product_detail .o_add_wishlist_dyn',
+        },
+        {
+            content: "go to wishlist",
+            extra_trigger: ".label:contains(2)",
+            trigger: 'a[href="/shop/wishlist"]',
+        },
+        {
+            content: "remove Ipad",
+            trigger: 'tr:contains("iPad Retina Display") a.o_wish_rm:first',
+        },
```

Comment:
idem here and next...

You need to check that the add to wishlist make the job, and e.g. that b2b let the item into the wishlist, ...

Here that just check that lcick don't crash, but the behaviour is not flow could be wrong